github.sgnl.yaml

```diff
@@ -640,6 +654,579 @@ entities:
       - name: viewerIsFollowing
         externalId: viewerIsFollowing
         type: Bool
+  
+  # https://docs.github.com/en/graphql/reference/objects#pullrequest
+  PullRequest:
+    displayName: PullRequest
+    externalId: PullRequest
+    description: Entity representing a GitHub PullRequest
+    pageSize: 100
+    pagesOrderedById: false
+    attributes:
+      - name: id
+        externalId: id
+        type: String
+        indexed: true
+        uniqueId: true
+        attributeAlias: pullRequestId
+      - name: baseRepositoryId
+        externalId: $.baseRepository.id
+        type: String
+        indexed: true
+        attributeAlias: baseRepositoryId
+      - name: headRepositoryId
+        externalId: $.headRepository.id
+        type: String
+        indexed: true
+        attributeAlias: headRepositoryId
+      - name: authorLogin
+        externalId: $.author.login
+        type: String
+        indexed: true
+        attributeAlias: authorLogin
+      - name: authorAssociation
+        externalId: authorAssociation
+        type: String
+      - name: baseRefName
+        externalId: baseRefName
+        type: String
+      - name: canBeRebased
+        externalId: canBeRebased
+        type: Bool
+      - name: changedFiles
+        externalId: changedFiles
+        type: Int64
+      - name: closed
+        externalId: closed
+        type: Bool
+      - name: closedAt
+        externalId: closedAt
+        type: DateTime
+      - name: createdAt
+        externalId: createdAt
+        type: DateTime
+      - name: createdViaEmail
+        externalId: createdViaEmail
+        type: Bool
+      - name: fullDatabaseId
+        externalId: fullDatabaseId
+        type: Int64
+      - name: deletions
+        externalId: deletions
+        type: Int64
+      - name: headRefName
+        externalId: headRefName
+        type: String
+      - name: includesCreatedEdit
+        externalId: includesCreatedEdit
+        type: Bool
+      - name: isCrossRepository
+        externalId: isCrossRepository
+        type: Bool
+      - name: isDraft
+        externalId: isDraft
+        type: Bool
+      - name: isInMergeQueue
+        externalId: isInMergeQueue
+        type: Bool
+      - name: isMergeQueueEnabled
+        externalId: isMergeQueueEnabled
+        type: Bool
+      - name: lastEditedAt
+        externalId: lastEditedAt
+        type: DateTime
+      - name: locked
+        externalId: locked
+        type: Bool
+      - name: mainterCanModify
+        externalId: mainterCanModify
+        type: Bool
+      - name: mergeStateStatus
+        externalId: mergeStateStatus
+        type: String
+      - name: mergeable
+        externalId: mergeable
+        type: String
+      - name: merged
+        externalId: merged
+        type: Bool
```

Comment:
Should we index this? 
e.g. "Deny access to PROD if PR is merged".

Reply:
added